source/renderer/webpack.config.js

```diff
@@ -24,8 +29,7 @@ module.exports = {
         include: /source/,
         exclude: /source\/main/,
         use: {
-          loader: 'babel-loader?cacheDirectory&cacheIdentifier=' + Math.random(),
-          // Fix for https://github.com/yahoo/babel-plugin-react-intl/issues/47 ^^
```

Comment:
@DominikGuzei this is the part which broke the intl setup...

Reply:
Perhaps we could put this as an integral part of the ‘npm run manage-translations’ task?

Reply:
I just pushed a fix: translations are never touched except when you run `npm run manage-translations` (then they are purged, cache cleared, build + regenerated)

Reply:
Thanks Dominik!
Will check it out in the morning ;)

Reply:
Thanks Dominik!
Will check it out in the morning ;)

Reply:
Works like charm 👍